### PR TITLE
Setup webpack and vscode build tasks

### PR DESCRIPTION
**Pull request by akaroml**

### CODE REVIEW

File: src/welcome/index.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    vscode.ViewColumn.One,
+    {
+      enableScripts: true,
+      enableCommandUris: true
+    }
+  );
+
+  readFile(require.resolve('./assets/index.html'), (err, data) => {
+    welcomeView.webview.html = data.toString();
+  }) ;
```
Comment (testforstephen) [inline src/welcome/index.ts]: extra whitespace before `;`

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+var HtmlWebpackPlugin = require("html-webpack-plugin");
```
Comment (yaohaizh) [inline webpack.config.js]: var ==> const

File: src/welcome/index.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import * as vscode from "vscode";
```
Comment (yaohaizh) [inline src/welcome/index.ts]: mixed of " & '

Comment (testforstephen) [inline src/welcome/index.ts]: `"` -> `'`. Keep code style consistent.

**Comment by akaroml** [comment]
Changed from double quotes to single quotes

**Pull request merged**